setup.cfg

```diff
@@ -4,9 +4,14 @@ exclude=webapp-django/crashstats/*/migrations/*,docs/conf.py,socorro/unittest/te
 max-line-length = 100
 
 [tool:pytest]
-addopts = -rsxX --tb=native
+# -rsxX        - show skipped, failed, and passed tests
+# --tb=native  - print native traceback
+# -p no:django - disable the pytest-django plugin for Socorro tests
+addopts = -rsxX --tb=native -p no:django
```

Comment:
This file (`setup.cfg`) is at the root of the repository. That's the directory you sit in when running pytest on the unit tests located in `socorro/unittest/`.

We've added pytest-django to the Python environment, but really don't want it loading when we run the regular Socorro tests. We only want it loading when running the webapp tests. So adding this `-p` option prevents it from loading.

While doing that, I figured I'd add some descriptive text about the other two options we do.